docs/web/examples/index.rst

```diff
@@ -9,8 +9,9 @@ Examples
 twisted.web.client
 ------------------
 
-- :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
-- :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
+- :download:`httpclient.py` - use ``twisted.web.client.Agent`` to download a web page.
+- (deprecated) :download:`getpage.py` - use ``twisted.web.client.getPage`` to download a web page.
+- (deprecated) :download:`dlpage.py` - add callbacks to ``twisted.web.client.downloadPage`` to display errors that occur when downloading a web page
```

Comment:
If you delete it, folks maintaining software written against Twisted X.Y can't easily find documentation for the code they're maintaining anymore.  Please leave the docs until the actual implementation goes away.  Certainly, make it clear that they're docs for a deprecated API until it is deleted.